docs/source/api/plugin/usage-reporting.md

```diff
@@ -1,20 +1,22 @@
 ---
-title: "API Reference: ApolloServerPluginUsageReporting"
-sidebar_title: ApolloServerPluginUsageReporting
+title: "API Reference: Usage reporting plugin"
+sidebar_title: "Usage reporting plugin"
 api_reference: true
 ---
 
-## Using the plugin
+Apollo Server has a built-in usage reporting plugin that gathers metrics on how your clients use the operations and fields in your GraphQL schema. The plugin also handles pushing these metrics to [Apollo Studio](https://www.apollographql.com/docs/studio/), as described in [Metrics and logging](../../monitoring/metrics/).
```

Comment:
I've been trying to move away from the term "metrics" for our usage reporting. Metrics feels very timing/performance-connected and I'm trying to emphasize "what clients use what fields" important rather than purely "and how long did it take to run".